GroupBy#6.5.py

```diff
@@ -0,0 +1,8 @@
+def group_by(l, args):
+    dic = {}

```

Comment:
replace 'l' and 'dic' Variable names - https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing